src/transformers/utils/modeling_auto_mapping.py

```diff
@@ -36,3 +36,323 @@
         ("IBertConfig", "IBertForQuestionAnswering"),
     ]
 )
+
+
+MODEL_FOR_CAUSAL_LM_MAPPING_NAMES = OrderedDict(
```

Comment:
I wonder if it wouldn't have been simpler (and prevented from having an additional list to maintain, even if you added the correct script modification) to have something like the following:
```py
MODEL_FOR_CAUSAL_LM_MAPPING_NAMES = {
    config.__name__: model.__name__ for config, model in MODEL_FOR_SEQUENCE_CLASSIFICATION_MAPPING.items()
}
```
(to adapt for tuples)

Reply:
No, the main point is to avoid importing that constant which imports every model.

Reply:
Can address further in a next PR if need be, I need to merge this to unlock the next steps :-)